plugins.go

```diff
@@ -0,0 +1,23 @@
+package gojenkins
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+//InstallPlugin installs a given plugin to jenkins instance
+func (jenkins *Jenkins) InstallPlugin(name, version string) error {
+
+	payload := `<jenkins><install plugin="` + name + `@` + version + `" /></jenkins>`
+
+	xmlBody := strings.NewReader(payload)
+
+	var body interface{}
+	err := jenkins.postXml("/pluginManager/installNecessaryPlugins", url.Values{}, xmlBody, &body)
+	if err != nil {
+		return err
+	}
+	fmt.Println(body)
```

Comment:
Again, we shouldn't print the response to the stdout.

Reply:
removed.